plugin.video.videolina/addon.py

```diff
@@ -0,0 +1,10 @@
+import xbmc
+import xbmcgui
+import xbmcplugin
+import xbmcaddon
+import logging
+from operator import itemgetter
```

Comment:
`xbmcplugin`, `xbmcaddon`, `logging` and `itemgetter` aren't used, so you don't have to import them (unless I am missing something).
I'd be fine merging it but just so you know for next time ;)